data/deep missions.txt

```diff
@@ -0,0 +1,1121 @@
+	to offer
+		"random" < 40
+		"deep convoy" >= 1
+	on offer
+		conversation
+			``
+
+# Start by walking through the spaceport and the player hears a TMBR song.
+# Look where it's coming from and find a bunch of Deep scientists rocking out to it while they board a Scout or Bounder (from the escort science vessel job).
+# Walk up to scientists and ask them about TMBR
+# One of them talks about how they couldn't go to the concert that happened in Syndicate space, but the one on the Lovelace planet was awesome
+# Another talks about how they haven't been spotted for days/weeks/months
+# The player can say how they know where TMBR are.
+# A required string
+# Land on a Deep planet and hear TMBR playing; scientists ask you to find them and bring them to Asgard where the scientists will be traveling
+
+# Characters: Maria, Garrison, Isaac, Hannah, Laura
+mission "Deep: TMBR 0"
+	minor
+	name ``
+	description ``
+	source
+		near Naos 1 100
+		attributes deep
+	destination Allhome
+	to offer
+		random < 60
+		"mystery cube" >= 10
+		"Transport scientists: done" >= 3
+		"Escort science vessel: done" >= 3
+		"Record academic conference: done" >= 3
+		"There Might Be Riots part 3B: done"
+	on offer
+		conversation
+			``
+
+# Land on Allhome where you left TMBR. Ask around and find out that TMBR are chilling on a different planet
+mission "Deep: TMBR 1"
+	landing
+	name ``
+	description ``
+	destination
+		distance 2 10
+		government Hai
+	to offer
+		"Deep: TMBR 0: done"
+	on offer
+		conversation
+			``
+				accept
+
+# Pick up TMBR and bring them to Asgard where the scientists are currently at. The scientists thank you for allowing them to listen to TMBR live
+mission "Deep: TMBR 2"
+	landing
+	name ``
+	description ``
+	destination Asgard
+	to offer
+		"Deep: TMBR 1: done"
+	on offer
+		conversation
+			``
+				accept
+
+	on complete
+		payment 400000
+		conversation
+			``
+
+###########################################################
+###########################################################
+###########################################################
+
+# The player gets called into somewhere and is asked about their work for Albert. They get questioned on what Albert told them, and seeing as how the player has been helping the Deep lately, the Deep tell the player some of their folklore.
+# "Over the centuries the exact date of when the Deep was established have been lost, but folklore says that the first humans arrived here in the mid 1300s before the time of the Black Death."
+mission "Deep: Singularity"
+	minor
+	name ``
+	description ``
+	source 
+	to offer
+		"deep convoy" >= 2
+		has "Deep: Mystery Cubes: done"
+		has "Deep: TMBR 2: done"
+	on offer
+		conversation
+			``
+
+mission "Deep: Remnant 0"
+	minor
+	name `Fetching Keystones`
+	description `Buy 10 Quantum Keystones from Hai space and return them to <destination> so that Ivan may research them and their link to the spacial anomaly in Terminus.`
+	source
+		attributes deep
```

Comment:
Can offer on Valhalla, which makes the bit where Ivan leaves for his lab on Valhalla odd (since you'd logically go there otherwise). Probably a `distance / near` filter with min 1 max 100 as well as the `attributes deep`, so that it can't offer on Valhalla.